client/src/components/ratingsReviews/ReviewTile.jsx

```diff
@@ -9,9 +9,9 @@ const ReviewTile = (props) => {
 
   let key = 0;
 
-  return reviews.length ? (
+  return reviews ? (
```

Comment:
My question here would be whether this component needs all of the reviews. Intuitively, one review tile would render one review.

Reply:
Yes, you are correct that a reviewTile would render one review. 
With this conditional render I'm checking whether or not reviews exist for the selected product. If they do, I map out the reviews array and assign a review tile to each of the individual reviews. Now that I'm thinking about it, I could probably lift this conditional render to reviewsList component and still map out the array inside the reviewTile.